commands/command_output.py

```diff
@@ -0,0 +1,21 @@
+from commands.status import Status
+
+class CommandOutput:
```

Comment:
I just explained this to Yehya, could you please look at Yehya repository. It can be cleaner and easier